case-uco/java/src/org/sleuthkit/caseuco/UcoObject.java

```diff
@@ -27,6 +27,8 @@
  */
 abstract class UcoObject {
 
+    static final String UCO_OBJECT = "uco-observable:";
```

Comment:
I'd name this something like UCO_OBSERV.  I think we're going to have several of these prefixes.

Reply:
Changed and commited